tests/queryset_pickle/tests.py

```diff
@@ -66,6 +66,20 @@ def test_model_pickle(self):
         self.assertEqual(original, reloaded)
         self.assertEqual(original.somefield, reloaded.somefield)
 
+    def test_model_pickle_with_ignoring_fields(self):
```

Comment:
Model pickling is tested in `model_regress/test_pickle.py`. This file is for queryset pickling.